src/client/dfs/dfs.c

```diff
@@ -3796,7 +3796,7 @@ dfs_removexattr(dfs_t *dfs, dfs_obj_t *obj, const char *name)
 	cond = DAOS_COND_DKEY_UPDATE | DAOS_COND_PUNCH;
 	rc = daos_obj_punch_akeys(oh, th, cond, &dkey, 1, &akey, NULL);
 	if (rc) {
-		D_ERROR("Failed to punch extended attribute %s\n", name);
+		D_INFO("Failed to punch extended attribute '%s'\n", name);
```

Comment:
should this use D_CDEBUG(rc == -DER_NONEXIST, DLOG_INFO, DLOG_ERR, "Failed to punch....);   Or should we use a debug bit?

Reply:
I wasn't aware of D_CDEBUG, that would be perfect for this.  Will update.